[NETOBSERV] OSDOCS-15503 Vale updates for network-observability-loki-secret.adoc (#96559)
openshift/openshift-docs: modules/network-observability-loki-secret.adoc

[id="network-observability-loki-secret_{context}"]
= Creating a secret for Loki storage

> The {loki-op} supports a few log storage options, such as AWS S3, Google Cloud Storage, Azure, Swift, Minio, {rh-storage}. The following example shows how to create a secret for AWS S3 storage. The secret created in this example, `loki-s3`, is referenced in "Creating a LokiStack resource". You can create this secret in the web console or CLI.

Reviewer: Please add a link in the Additional resources section for "Creating a LokiStack resource" document as I don't see it in the book.

> . Using the web console, navigate to the *Project* -> *All Projects* dropdown and select *Create Project*. Name the project `netobserv` and click *Create*.

Reviewer: Might be out of scope, but consider stepping out "Name the project `netobserv` and click *Create*." in its own step as step 1 has a few steps.

<1> The installation examples in this documentation use the same namespace, `netobserv`, across all components. You can optionally use a different namespace for the different components

.Verification

> * Once you create the secret, you should see it listed under *Workloads* -> *Secrets* in the web console.

Reviewer: ```suggestion
* After you create the secret, you can view the secret under *Workloads* -> *Secrets* in the web console.
```

"Should" does not provide 100 % confidence, IMO, that the secret will be there after completing the procedure.
